labelbox/client.py

```diff
@@ -1385,6 +1392,27 @@ def get_catalog_slice(self, slice_id) -> CatalogSlice:
         res = self.execute(query_str, {'id': slice_id})
         return Entity.CatalogSlice(self, res['getSavedQuery'])
 
+    def unarchive_feature_schema_node(self, ontology_id: str,
+                                      root_feature_schema_id: str) -> bool:
+        """
+        Returns true if the root feature schema node was successfully unarchived, false otherwise
+        Args:
+            root_feature_schema_id (str): The ID of the root level feature schema
+            ontology_id (str): The ID of the ontology
+        Returns:
+            bool
+        """
+        ontology_endpoint = self.rest_endpoint + "/ontologies/" + ontology_id + '/feature-schemas/' + root_feature_schema_id + '/unarchive'
+        response = requests.patch(
+            ontology_endpoint,
+            headers=self.rest_endpoint_headers,
+        )
+        if response.status_code == 200:
+            return response.json()['unarchived']
+        else:
+            raise labelbox.exceptions.LabelboxError(
+                "Failed unarchive root feature schema node: ", response.text)
```

Comment:
This is the convention other methods follow

```suggestion
                "Failed unarchive root feature schema node, message: ", response.text)
```

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/939/commits/4764f3e85dc2ce7fcff2f038389fd8790c392af2)